src/NzbDrone.Common/Extensions/UrlExtensions.cs

```diff
@@ -11,6 +11,11 @@ public static bool IsValidUrl(this string path)
                 return false;
             }
 
+            if (path.StartsWith(" ") || path.EndsWith(" "))
```

Comment:
This looks a bit funky. Shouldn't it be any space in the URL rather than just start and end? I'm not sure what this code attaches to, but would it be worth it to just strip whitespace?

Reply:
White space in the url should get encoded